packages/canary-web-components/src/components/ic-menu-with-multi/ic-menu.tsx

```diff
@@ -1394,16 +1394,16 @@ export class Menu {
     return (
       <Host
         class={{
-          "full-width": fullWidth,
+          "ic-menu-full-width": fullWidth,
           "no-focus":
```

Comment:
should maybe change "no-focus" too, as still quite a generic name that might clash?